.github/workflows/ci.yaml

```diff
@@ -22,15 +22,20 @@ jobs:
     steps:
       - name: Run checkout
         uses: actions/checkout@v4
+
       - name: pester
         run: Invoke-Pester -Show All
-      - name: Install go
```

Comment:
Is it true that GoLang is available in the runner by default? 
Is it the same as the one in the go.mod file?

Reply:
Go is [installed by default on windows runners](https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md), and the use of the `go toolchain` statement in `go.mod` results in [CI downloading the proper version of Go](https://github.com/rancher/permissions/actions/runs/11016785328/job/30593135049?pr=2) when running the `go test` step. I've found that the install go step in CI can some times take several minutes on Windows runners. This is a bigger problem in the wins repository right now, and I'd like to avoid it here. Now that `go toolchain` automatically downloads the correct version, this step isn't really needed.

Reply:
Thanks for the explanation.  It makes sense to me.